java/src/test/java/com/jongallant/azure/identity/extensions/DefaultAzureServiceBusCredentialTest.java

```diff
@@ -0,0 +1,27 @@
+package com.jongallant.azure.identity.extensions;
+
+import com.azure.identity.DefaultAzureCredential;
+import com.microsoft.azure.servicebus.ClientSettings;
+import com.microsoft.azure.servicebus.Message;
+import com.microsoft.azure.servicebus.TopicClient;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit test for {@link DefaultAzureServiceBusCredential}.
+ */
+public class DefaultAzureServiceBusCredentialTest {
+
+    /**
+     * Unit test to use {@link DefaultAzureCredential} to create a {@link TopicClient} and send a message.
+     * @throws ServiceBusException
+     * @throws InterruptedException
+     */
+    @Test
+    public void testDefaultCredential() throws ServiceBusException, InterruptedException {
+        TopicClient topicClient = new TopicClient("jongsb2", "srnagartopic",
```

Comment:
Please implement the Environment variable scheme from .NET version, read in the .env file that is in the root.  Thanks.

Reply:
Reading from environment file in classpath